### ISSUE TITLE
Deployed via Heroku, No map?

### ISSUE THREAD

**Issue by boltron88** [AHAAAAAAA/PokemonGo-Map#2110]
Hello, 
I have deployed this via Heroku, but i am unable to find the map O.o
i get all of the settings, but dont see how to access the actual map
Any help would be fantastic

**Comment by spiritman110** [comment]
Yeah it looks like the css is missing for some reason as well, I don't get a map either.

**Comment by swtrpeter** [comment]
I wish there was some documentation on deploying this to heroku. I would really like to try that route but have no idea where to start.

**Comment by bsphil** [comment]
The most recent commit,

**Comment by M0rr1son** [comment]
Having the same issue as @spiritman110, first time using Heroku and don't really know what @bsphil is suggesting.

**Comment by Kostecki** [comment]
You can do a rollback on heroku using `heroku releases` - find a working version and then `heroku rollback` to that version.

**Comment by boltron88** [comment]
Works with latest Dev branch

**Comment by hack3rjack** [comment]
Dev and Travis branches (from a few hours ago) works on FF and Chrome for me, but IE on Heroku is broken AFAIK

**Comment by GreenflyM** [comment]
Dev branch working now, but only on desktop. Mobile devices see a blank screen.
